src/main/java/com/stripe/model/Charge.java

```diff
@@ -602,6 +607,487 @@ public void setRuleObject(Rule rule) {
     }
   }
 
+  @Getter
+  @Setter
+  @EqualsAndHashCode(callSuper = false)
+  public static class PaymentMethodDetails extends StripeObject {
+    @SerializedName("account")
+    Account account;
```

Comment:
I think the field name should be `stripe_account` instead?
And we can consider renaming the class below to `StripeAccount`?